pkgpanda/build/cli.py

```diff
@@ -5,8 +5,8 @@
 the necessary dependencies.
 
 Usage:
-  mkpanda [--repository-url=<repository_url>] [--dont-clean-after-build] [--recursive]
-  mkpanda tree [--mkbootstrap] [--repository-url=<repository_url>] [<variant>]
+  mkpanda [--repository-url=<repository_url>] [--dont-clean-after-build] [--recursive] [--variant=<variant>]
+  mkpanda tree [--mkbootstrap] [--repository-url=<repository_url>] [--variant=<variant>]
```

Comment:
Because then mkpanda tree is borked forever as when you do:
`mkpanda tree` it thinks you are doing mkpanda with variant=tree. Its either do it this way or change the form of the command to be something like `mkpanda package` and `mkpanda tree` so that way a command could never be parsed as an argument by not having a command `mkpanda` that takes a positional arg